app/models/facility.rb

```diff
@@ -43,6 +43,10 @@ def accounts
   validates_format_of    :abbreviation, with: /^[a-zA-Z\d\-\.\s]+$/, message: "may include letters, numbers, hyphens, spaces, or periods only"
   validates_format_of    :journal_mask, with: /^C\d{2}$/, message: "must be in the format C##"
 
+  validates :order_notification_recipient,
+    format: { with: /^[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,6}$/i },
```

Comment:
I'm not sure I want to pull in a dependency for this, but I do think standardizing our email validation is a good idea. For the training request notification feature, I did minimal validation https://github.com/tablexi/nucore-open/blob/master/app/models/product.rb#L31

For public sites, I definitely lean towards lenient email validation to allow that random person with `☃@snowman.com`, but in NUcore's case, I'd be fine with continuing with this regex.